### DISCUSSION TITLE
Are both my CC2531 broken

### DISCUSSION THREAD

**Discussion by kimmonnens** [Koenkk/zigbee2mqtt#12835] [General]
My zigbee2mqtt setup with a CC2531 adapter was running without issues for more than a year on a PI4.
The PI4 got broken so I bought a new PI4 with a new SD, installed everything from scratch and now pairing is not working anymore.
I have tried multiple versions of zigbee2mqtt and also flashed firmwares from 2019, 2020 and 2021 on the CC2531. I also tried all the same with my 2nd  CC2531 adapter, but pairing is not working.

To make sure that it was not related to the PI4, i did a clean install on my Windows PC. Also tried with both my CC2531 adapters and the 3 different firmwares. Same result pairing is not working.

There is a 60cm USB cable between the adapter and the PI4/Windows PC. I have tried with and without the USB cable.

Devices I tried to pair: IKEA remote and IKEA bulbs. When trying to pair, they are located next to the adapter.

Below is my last configuration file and logging. Nothing related to pairing in the logging.

So I wonder if both my CC2531 adapters are broken or I am doing something wrong. Any advice is welcome.

Thank you.

[configuration.txt](https://github.com/Koenkk/zigbee2mqtt/files/8912094/configuration.txt)
[log.txt](https://github.com/Koenkk/zigbee2mqtt/files/8912096/log.txt)

**Comment by kimmonnens** [comment]
I have ordered the Slaesh's CC2652RB stick and waiting for it to arrive.
Meanwhile I am trying to find out if the CC2531 is broken.
Attached the logging with herdsman debug activated.
Maybe someone can see in the logging what is going wrong?

Thank you.

[log_and_herdsman.txt](https://github.com/Koenkk/zigbee2mqtt/files/8933107/log_and_herdsman.txt)

**Comment by Koenkk** [comment]
Your CC2531 looks fine, but make sure to power down your complete old network before trying to setup a new one.

**Comment by kimmonnens** [comment]
@Koenkk 
Thank you, you were correct. I missed one TRADFRI signal repeater that was still plugged-in. After unplugging the pairing of all devices was working again.
